themes/docs-new/layouts/shortcodes/workstation.md

```diff
@@ -1,8 +1,10 @@
-Chef Workstation includes:
+<p>Chef Workstation includes:</p>
```

Comment:
Undo the changes to this file. It should be in MD.

Reply:
List doesn't render in markdown

Reply:
I just tested it, it works fine for me.